src/assets/routes.ts

```diff
@@ -0,0 +1,40 @@
+import Home from '../components/Home';
+import About from '../components/About';
+import PyBuild from '../components/Tools/pybuild';
+import CMDWord from '../components/Tools/cmd-word';
+import DTEFileCreator from '../components/Tools/dte-file-creator';
+import MemMap from '../components/Tools/memmap';
+import MissShob from '../components/Tools/miss-shob';
+
+const routes = [
```

Comment:
(Not critical, but VERY good practice):

Notice that `react-router` has `RouteProps` as a named export, which you can improt.

You can define `routes` to be of type `RouteProps[]`, and then have better control over which elements you pass to each "link"

(Of course, if you do that, you'd have to change `link` to `path` and `comp` to `component`, but it would make more sense to anyone new reading this code)